src/styles/global.scss

```diff
@@ -8,10 +8,17 @@
 
 * {
   box-sizing: border-box;
+  margin: 0px;
+  font-family: "Source Code Pro", monospace, sans-serif;
```

Comment:
Should fonts also be defined in a MUI theme (that light and dark inherit from)?

Reply:
Yes, font should be defined on the theme.ts.